src/components/Person/Person.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person, Sex } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const {
+    name, slug, sex, born, died, motherName, fatherName,
+  } = person;
+
+  const styleRed = classNames({
+    'has-text-danger': person.sex === Sex.Female,
+  });
+
+  const hasMother = people
+    .find(mother => mother.name === person.motherName);
+  const hasFather = people
+    .find(father => father.name === person.fatherName);
```

Comment:
At the time of the search, you do not yet know whether it is `father` or `mother`
```suggestion
  const hasMother = people.find(({ name }) => name === person.motherName);
  const hasFather = people.find(({ name }) => name === person.fatherName);
```